include/hamlib/rig.h

```diff
@@ -93,24 +93,6 @@
 #endif
 
 #include <time.h>
```

Comment:
It seems like this line should have been moved to sleep.c as well and the local include of rig.h omitted.